test/specs/new/list_item_unindented_asterisk.md

```diff
@@ -0,0 +1,20 @@
+---
+pedantic: true
```

Comment:
Why pedantic? Pedantic is meant to follow the original markdown rules not commonmark.

Reply:
Oops. Sorry, must've slipped while poking here and there. Pushed fixes.